client/landing/stepper/declarative-flow/newsletter.ts

```diff
@@ -17,14 +17,7 @@ export const newsletter: Flow = {
 			recordTracksEvent( 'calypso_signup_start', { flow: this.name } );
 		}, [] );
 
-		return [
-			'intro',
-			'newsletterSetup',
-			'completingPurchase',
-			'processing',
-			'subscribers',
-			'launchpad',
-		] as StepPath[];
+		return [ 'intro', 'newsletterSetup', 'processing', 'subscribers', 'launchpad' ] as StepPath[];
```

Comment:
We can remove the processing step as part of the flow now.
It was only used by completingPurchase.